csharp/src/Ice/IPlugin.cs

```diff
@@ -9,6 +9,8 @@ namespace ZeroC.Ice
     public interface IPlugin : IAsyncDisposable
     {
         /// <summary>Perform any necessary initialization steps.</summary>
-        void Initialize();
+        /// <param name="context">The plug-in initialization context allows to register invocation and dispatch
+        /// interceptors, it can also be use to set the communicator logger.</param>
+        void Initialize(PluginInitializationContext context);
```

Comment:
```suggestion
        /// <param name="context">The plug-in initialization context enables the registration of invocation and dispatch
        /// interceptors, it can also be use to set the communicator logger.</param>
        void Initialize(PluginInitializationContext context);
```